Sbank 195 confirm registration

Parker-Williams-Smoothstack opened the pull request.

Parker-Williams-Smoothstack committed:
Add files via upload

Parker-Williams-Smoothstack committed:
Add files via upload

Parker-Williams-Smoothstack committed:
Add files via upload

Parker-Williams-Smoothstack committed:
Add files via upload

Parker-Williams-Smoothstack committed:
Minor tweaks to CardRegistration

szahedi-smoothstack commented on src/components/CardRegistration.js:
```diff
@@ -0,0 +1,305 @@
+        address: Joi.string().required().label("Mailing address"),
+        po_box: Joi.string().label("P.O box"),
+        zipcode: Joi.number().required().label("Zipcode"),
+        ssn: Joi.string().min(10).max(10).required().label("Social Security Number"),        
```
If I'm not mistaken, isn't SSN 9 digits?

Parker-Williams-Smoothstack replied:
SSN is nine digits, 3 area digits 2 Group Number digits and 4 Serial Number digits according to https://www.ssa.gov/history/ssn/geocard.html#:~:text=The%20nine-digit%20SSN%20is%20composed%20of%20three%20parts%3A,set%20of%20four%20digits%20is%20the%20Serial%20Number.

szahedi-smoothstack replied:
I suggest changing the phone numbers to 10 that way area code is always needed, and then ssn can change to min(9).max(9)

Parker-Williams-Smoothstack replied:
Made the changes, so it should now use the correct number of digits for SSN and the phone numbers now ask for the area code (10 digits up from 7 digits).

Parker-Williams-Smoothstack committed:
Correct SSN requirement

Parker-Williams-Smoothstack committed:
Phone number changes

All phone numbers now require at least 10 digits, meaning area codes are now a hard requirement.

szahedi-smoothstack reviewed (approved):
Looks good
